mm2src/coins/utxo.rs

```diff
@@ -1907,6 +1907,12 @@ pub fn output_script(address: &Address, script_type: ScriptType) -> Script {
     match address.addr_format {
         UtxoAddressFormat::Segwit => Builder::build_p2witness(&address.hash),
         _ => match script_type {
+            ScriptType::P2PK => Builder::build_p2pk(
+                address
+                    .hash
+                    .get_pubkey()
+                    .expect("`get_pubkey` is called on a non-PubKey address"),
```

Comment:
Do we consider panicking here as developer error? It's not impossible that someone will use an address with non-`PubKey` hash variant. cc @shamardy

Reply:
> Do we consider panicking here as developer error?

You are right, but I consider this part will change after this https://github.com/KomodoPlatform/komodo-defi-framework/pull/1960 is merged, ref. https://github.com/KomodoPlatform/komodo-defi-framework/pull/1960#discussion_r1475412426